docs/src/domains/cdn/_index.md

```diff
@@ -5,179 +5,168 @@ weight: 3
-These vendors include their own Level 3 DDoS protection that is sufficient for the majority of cases.
-
-Customers are welcome to put their own WAF in front of a Dedicated cluster or add other security measures not included in the offering.
-
-## The router cache
-
-When using a CDN the Platform.sh router's HTTP cache becomes redundant.
-In most cases it's best to disable it outright.
-Modify your route like so to disable the cache:
-
+ 
+Using a CDN speeds up the delivery of your site's content to its users.
+The CDN deploys edge servers at many locations around the world.
+These edge servers behave like local caches to nearby users.
+Bringing content closer to users helps enhance your site's perceived performance
+and so can improve user engagement and retention.
+
+Fastly is the recommended CDN for Platform.sh projects.
+By default, Dedicated projects include a [Fastly](./fastly.md) CDN managed by Platform.sh.
+Self-service Grid plans don't include a CDN by default, but you can set up one at any time.
+
+## CNAME records
+
+To start routing client traffic through your CDN,
+create `CNAME` records for your domain names through your DNS provider.
+For more information, see you DNS provider's official documentation.
+
+Note that `CNAME` records can't point to apex domains,
+but most CDN providers offer [workarounds](../steps/dns.md#handling-apex-domains).
+
+## DDoS protection
+
+In a distributed denial of service (DDoS) attack, 
+an attacker floods a server with fake requests.
+The resources hosted by that server become inaccessible to real users.
+
+All sites hosted by Platform.sh live on infrastructure provided by major cloud providers.
+By default, you benefit from the powerful DDoS protection services these cloud providers offer.
+In most cases, these protection services are enough to keep your site safe from DDoS attacks,
+including when you use a CDN.
+
+Enterprise and Elite projects on Platform.sh also include a [web application firewall (WAF)](../../security/waf.md). 
+Depending on your needs, you can install your own WAF and add security measures not included in your plan.  
+
+## Third-party TLS certificates
+
+Dedicated plans include two [transport layer security (TLS) certificates](../../other/glossary.md#transport-layer-security-tls), 
+an apex and a wildcard one.
+This allows for encryption of all traffic between your users and your app. 
+
+If you want to, you can also provide your own third-party TLS certificate.
+[Transfer your certificate](../../development/file-transfer.md), 
+its unencrypted private key and the intermediate certificate to a non-web accessible [mount](../../create-apps/app-reference.md#mounts), 
+on an environment that only Platform.sh support and trusted users can access.
+In this way, your private key can't be compromised.
+To add the TLS certificate to your CDN configuration, 
+[create a support ticket](../../overview/get-support.md#create-a-support-ticket).
```

Comment:
My understanding was that it was available to anyone provided they paid an additional fee for it. But it also seems that we don't want to encourage users to use their own TLS certificates as most of them then forget to renew them (see context). Might be a bold choice, but if the certificates we provide are identical and don't present any risk of botched renewal, maybe it's best to remove the section entirely to avoid giving any ideas to clients? What do you think?

Reply:
The additional context seems to be only about certificates for Fastly when it's managed by us, and not a generic CDN. Maybe there should be a note on the Fastly part, or within a Dedicated section. I wouldn't remove just it because people forget to renew. But not sure it's relevant here, sure.